azure-pipelines.yml

```diff
@@ -79,10 +79,22 @@ jobs:
     - script: sh -x ./clean_build.sh
       displayName: Clean Build
 
-- job: MacBuild
-  displayName: Mac Build
+- job: macOStests
+  displayName: macOS Tests
```

Comment:
If we're adding release build to this same matrix, let's keep the old names.

Reply:
Fixed in ed981a41.